src/directLine.ts

```diff
@@ -370,7 +370,8 @@ export interface DirectLineOptions {
     streamUrl?: string,
     timeout?: number,
     // Attached to all requests to identify requesting agent.
-    botAgent?: string
+    botAgent?: string,
+    conversationStartLocale?: string
```

Comment:
```suggestion
    localeOnStartConversation?: string
```

I think this name is better for developer to understand the locale is only for "Start Conversation" API/operation.

Reply:
Naming recommendation for "property bag":

- Per-user or per-agent (a.k.a. per-browser)
- One-off and immutable